src/graphql/resolvers.ts

```diff
@@ -44,8 +45,16 @@ export const resolvers = {
         throw new CustomError(401, 'Operação não autorizada.');
       }
 
+      if (args.data.numberOfUsers < 1 || args.data.skippedUsers < 0) {
+        throw new CustomError(400, 'Operação inválida');
+      }
+
+      const totalUsers = await prisma.user.count();
+      const skipped = args.data.skippedUsers || defaultSkippedUsers;
```

Comment:
```suggestion
      const skipped = args.data.skippedUsers ?? defaultSkippedUsers;
```
https://stackoverflow.com/questions/61480993/when-should-i-use-nullish-coalescing-vs-logical-or

e acho que dava pra colocar de default no schema do graphql, perhaps